release-notes/aws/v11.1.4.md

```diff
@@ -0,0 +1,22 @@
+# :zap: Giant Swarm Release v11.1.4 for AWS :zap:
+
+__Note:__ Upgrading to this release from any release prior v11.1.1 will cause a network downtime due to the network-related changes coming with the switch from Calico CNI to AWS CNI.
+
+## aws-operator [v8.2.x](https://github.com/giantswarm/aws-operator/releases/tag/v8.2.x)
+
+- Ensure that Pod subnet IP ranges don't overlap between tenant clusters, to enable VPC peering between tenant clusters in the same installation.
+
+## cert-manager-app [v1.0.7](https://github.com/giantswarm/cert-manager-app/releases/tag/v1.0.7)
+
+- Fixed sub-chart resources namespace. ([cert-manager-app#19](https://github.com/giantswarm/cert-manager-app/pull/19), [cert-manager#21](https://github.com/giantswarm/cert-manager-app/pull/21)).
+
+## chart-operator [v0.12.4](https://github.com/giantswarm/chart-operator/releases/tag/v0.12.4)
```

Comment:
@rossf7 I said I'd check w you that latest chart-operator gets into the releases. YAML file version is changed and the release notes are here - that's all I can check. Seems ok for me. Is that good enough for AWS 11.1.4? :)

Reply:
Oh. I saw your conversation with Marian here. https://github.com/giantswarm/giantswarm/issues/10019#issuecomment-615066947 Closing comment.